### ISSUE TITLE
Provide an edit method to move to the next or previous blank line with and without selection

### ISSUE THREAD

**Issue by euclio** [xi-editor/xi-editor#974]
# Xi-Editor Issue Template
- [x] I have searched existing issues and could not find my issue.
- [x] I have studied the documentation.

Such a method is needed to implement vi's `{` and `}` "paragraph" motions.

**Comment by cmyr** [comment]
Does `moveToEndOfParagraph` do what you want? The selection variants don't exist, but they could be added.

**Comment by euclio** [comment]
I don't think so, I think the RPC and the vim motion have different concepts of paragraph. The vim motion will essentially move to the next blank line, while I think the RPC will move to the new newline?
